lib/puppet/type/cs_order.rb

```diff
@@ -59,9 +59,9 @@
       of multiple order groups and so there is a way to control which
       primitives get priority when forcing the order of state changes on
       other primitives.  This value can be an integer but is often defined
-      as the string INFINITY."
-
-    defaultto 'INFINITY'
+      as the string INFINITY.
+      When using pcs as provider this value is not used. It should be preferred
+      to use parameter kind in general."
```

Comment:
i think this wording is better:
```suggestion
      When using pcs as provider this value is not used.
      It is generally preferred to use the `kind` parameters."
```

Reply:
well that worked great…
anyway, don't click merge this patch, please, but i hope you get the idea

Reply:
> well that worked great…
> anyway, don't click merge this patch, please, but i hope you get the idea

Well, i don't have permissions to do it anyway. :) And thanks for the comments. I am leaving the resolving of the conversations, where i did not change it on purpose and explained why, up to you to close it. As well as this one due to this feedback here.

Reply:
Closing conversation, since i changed the wording and you approved.